community/concepts/cred.md

```diff
@@ -0,0 +1,18 @@
+# Cred
+
+SourceCred, as the name might suggest, is all about attributing cred. Cred is a metric that describes every contribution and contributor in a project to give a sense of how important they were.
+
+For example, forum posts can earn cred. If I post on the [sourcecred discourse](https://discourse.sourcecred.io/) and earn 5 cred, and another user posts and their post earns 10 cred, we can say that the other post is considered twice as important as my post.
+
+The contributions are arranged in a [graph](https://en.wikipedia.org/wiki/Graph_(discrete_mathematics)) where contributions are nodes, and have edges indicating how they relate to other contributions. Contributors — like you or me — are also nodes in the graph, and are connected to the contributions that they create. So there is an "AUTHORS" edge connecting me (the author) to my post. As a more complete example, if someone writes a reply to my post, there will be an edge connecting the reply and my post, and I am also represented as a node
```

Comment:
I do believe it's called an authors edge (without the ') as @decentralion mentioned this in one of the plugin READMEs and I started to correct to add an apostrophe, but then realized it might actually be called that (without it).

Reply:
And I'm very happy to make your suggested change!

Reply:
Done.